calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/CalligraphyConfig.java

```diff
@@ -131,6 +138,15 @@ public int getAttrId() {
         return mAttrId;
     }
 
+    /**
+     *
+     * @param viewClass class to be ignored by Calligraphy
+     * @return true if class should not be injected, false otherwise
+     */
+    public boolean isIgnoredClass(Class<?> viewClass) {
```

Comment:
@chrisjenx where should I store collection of ignored classes if it will be in utils?